patches/chrome-android-java-AndroidManifest.xml.patch

```diff
@@ -66,6 +66,24 @@ index 5b5fe60a41618735c6bc44d50af8c6434ec28740..1b3536b2b40a1b2cdf7b0512931d1b6e
              <meta-data android:name="android.app.searchable"
                  android:resource="@xml/searchable" />
          </activity-alias>
+@@ -889,7 +903,7 @@ by a child template that "extends" this file.
+         </service>
+ 
+         <!-- Quick Action Search widget -->
+-        <receiver
++        <!-- <receiver
```

Comment:
It's generally better to add new line for direct patches, instead of modifying existing one.